pandas/tests/io/test_excel.py

```diff
@@ -129,32 +129,34 @@ def test_usecols_list(self, ext):
         dfref = self.get_csv_refdf('test1')
         dfref = dfref.reindex(columns=['B', 'C'])
         df1 = self.get_exceldf('test1', ext, 'Sheet1', index_col=0,
-                               usecols=[0, 2, 3])
```

Comment:
why are all of these changing?

Reply:
Because the last changes you required in the code implied in removing backward compatibility for the `index_col` keyword behavior of `read_excel`.

@jreback, take a look here to understand better what I'm meaning: https://github.com/pandas-dev/pandas/pull/20480#discussion_r187773244